elements/pl-file-preview/pl-file-preview.py

```diff
@@ -1,22 +1,15 @@
-import base64
-
 import chevron
-import lxml.html
 import prairielearn as pl
 
 
-def prepare(element_html, data):
-    element = lxml.html.fragment_fromstring(element_html)
-    required_attribs = []
-    optional_attribs = []
-    pl.check_attribs(element, required_attribs, optional_attribs)
-
-
-def render(element_html, data):
+def render(element_html: str, data: pl.QuestionData):
     if data["panel"] != "submission":
         return ""
 
-    html_params = {"uuid": pl.get_uuid()}
+    html_params = {
+        "uuid": pl.get_uuid(),
+        "submission_files_url": data["options"]["submission_files_url"],
+    }
 
     # Fetch the list of required files for this question
     required_file_names = data["params"].get("_required_file_names", [])
```

Comment:
No need to make this a variable if used once.
```suggestion
    html_params["required_files"] = data["params"].get("_required_file_names", [])
```